zombies.js

```diff
@@ -158,7 +196,20 @@
  * @name equip
  * @param {Weapon} itemToEquip  The weapon item to equip.
  */
-
+Player.prototype.equip = function(itemToEquip) {
+  if (this.getPack().indexOf(itemToEquip) !== -1 && itemToEquip instanceof Weapon) {
+    this.getPack().splice(this.getPack().indexOf(itemToEquip), 1, this.equipped);
+    this.equipped = itemToEquip;
+    console.log("You did the ole switcharoo with your weapons");
+    return false;
+  } else if ( this.getPack().indexOf(itemToEquip) === -1 && !(itemToEquip instanceof Weapon)) {
```

Comment:
Check your logic here. I think this allows you to equip a food time and says it's a weapon.